changelogs/unreleased/6148-lubronzhan-small.md

```diff
@@ -0,0 +1 @@
+Add anti-affinity rule for envoy deployed by provisioner
```

Comment:
lets bump this to a minor release note and note the change to the Envoy Deployment anti-affinity configuration in the default YAMLs as well just so people are aware if theyre using it